tests/controls/listctrltest.cpp

```diff
@@ -151,6 +154,37 @@ void ListCtrlTestCase::ColumnClick()
 
     m_list->ClearAll();
 }
+
+void ListCtrlTestCase::SubitemRect()
+{
+    wxImageList *m_imageListNormal = new wxImageList(32, 32, true);
+#ifdef wxHAS_IMAGES_IN_RESOURCES
+    m_imageListNormal->Add( wxIcon( "sample", wxBITMAP_TYPE_ICO_RESOURCE ) );
+#else
+    m_imageListNormal->Add( wxIcon( "sample.xpm" ) );
+#endif
+    m_list->InsertColumn(0, "Column 0", wxLIST_FORMAT_LEFT, 60);
+    m_list->InsertColumn(1, "Column 1", wxLIST_FORMAT_LEFT, 60);
+    m_list->InsertColumn(2, "Column 2", wxLIST_FORMAT_LEFT, 60);
+    for ( int i = 0; i < 3; i++ )
+    {
+        long tmp1 = m_list->InsertItem(i, wxString::Format("This is item %d", i), 0);
```

Comment:
This doesn't matter very much, but who calls variables `tmp1` (especially when there is no `tmp` or `tmp0`) rather than `index` or `n` or just using `i`?